### ISSUE TITLE
Incorrect lint error for valid formatting

### ISSUE THREAD

**Issue by orlov-vo** [prettier/eslint-plugin-prettier#279]
**What version of `eslint` are you using?**
`6.5.1`

**What version of `prettier` are you using?**
`2.0.2`

**What version of `eslint-plugin-prettier` are you using?**
`3.1.2` 

**Please paste any applicable config files that you're using (e.g. `.prettierrc` or `.eslintrc` files)**
```json
{
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "all",
  "semi": false,
  "arrowParens": "avoid"
}
```

**What source code are you linting?**

```js
export function prepareString(str) {
  return (str || '').toLocaleLowerCase().split(/\s/g).filter(Boolean)
}
```

**What did you expect to happen?**
No errors from `eslint`

**What actually happened?**
I've got an error from `eslint(prettier/prettier)`:

> Replace `.toLocaleLowerCase().split(/\s/g)` with `⏎····.toLocaleLowerCase()⏎····.split(/\s/g)⏎····`

**Comment by BPScott** [comment]
That's seems to be how [prettier 1.19.0 formats that content](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACAMwFcowYBLaPDbODAQ1oGUZtyoBzACgGdWBKPMAA6UPHlowi2Mb1Z4APgrwByFfwB0MCABkIYegBs4egO5xsAYXo84XTTwyHyMLgHphwnm46aC5Q3hsLgAhCAhjeih+UQBfEAAaEAgMCmgeZFBGbAhTAAVGBAyUegA3CHIAE0SQACNsejAAazgYJgYwdg5kViI4JIALGABbQwB1AZc4R0a4JiKXclKXAE9kcB4MpPZbXDyGjmH6ZAIjWySAKx40EIbm1qZ6YZN2OBOz-pArtCYu4wBFIgQeDvQznEAMbC7da1ei1OCGGo0dgwMZVGADZAADgADEkaBBbGMGhh1jRphZSm8kgBHIHwfYpYogGwAWigcDglS5NVodPItH29EOxyQpzBn1sw3IPWwfSSPD+cEBwLeSF6nxgcLRlQxyAATElWPQAl1LBBhkd1lBoNSQERbAAVOHFcXg0p9ACSUG5sCYYDYqQAgj6WCtjKDbLFYkA)

When formatting that code in a project using prettier 2.0.2 I get the expected formatting.

Can you check if you're not accidentally got two versions of prettier in your dependencies, and if you do try and deduplicate them - if you're using yarn you can do `yarn why prettier` to investigate

**Comment by orlov-vo** [comment]
@BPScott yes, you are right:

```
=> Found "prettier@2.0.2"
info Has been hoisted to "prettier"
=> Found "abstract-syntax-tree#prettier@1.19.1"
info This module exists because "rollup-plugin-amd#@buxlabs#amd-to-es6#abstract-syntax-tree" depends on it.
```

Do you know how we can fix it for me and other users with same situation?

**Comment by BPScott** [comment]
Try deleting your node_modules folder and recreating it - that might help make sure the folder structure is in the right shape after the change of prettier version so eslint picks up the right version of prettier.

If that doesn't work try updating your dependencies as it looks the newest version of abstract-syntax-tree no longer depends upon prettier, so try updating your dependency on rollup-plugin-amd to update its transitive dependencies.

If that doesn't work then if you're using `yarn` then you could use [resolutions](https://classic.yarnpkg.com/en/docs/selective-version-resolutions/) to force a specific prettier version.


I'm going to close this issue, as there's nothing to be done in eslint-plugin-prettier - we deliberatly leave depending on a version of prettier up to the consuming project. It's up to you to try and ensure you to make sure you provide the desired version of prettier.
